charts/kubernetes-agent/tests/tentacle-deployment-container-env-vars_test.yaml

```diff
@@ -374,7 +374,7 @@ tests:
         path: spec.template.spec.containers[0].env[?(@.name == 'OCTOPUS__K8STENTACLE__SCRIPTPODIMAGETAG')].value
         value: "1.0.0-deploymentTarget"
     - equal:
-        path: spec.template.spec.containers[0].env[?(@.name == 'OCTOPUS__K8STENTACLE__SCRIPTPODIMAGEPULLPOLICY')].value
+        path: spec.template.spec.containers[0].env[?(@.name == 'OCTOPUS__K8STENTACLE__SCRIPTPODPULLPOLICY')].value
```

Comment:
Is there also a test for the worker that uses the same variable?

Reply:
in the v2 chart yeah, will do that in the forward merge